README.md

```diff
@@ -61,6 +61,25 @@ upstream dependencies e.g. https://github.com/go-task/task/releases
 
 ---
 
+## Renovate Presets
+
+If you are using [Renovate](https://docs.renovatebot.com/) (for automated dependency updates) you can use/extend our Drupal presets by doing the following:
+
+```
+{
+  "extends": [
+    "github>Lullabot/drainpipe//renovate-presets/presets/drupal-10.json5"
+ ]
+}
+```
+
+This preset offers safe automation with flexibility and control for teams maintaining Drupal applications.
+It minimizes risk by requiring approval for major changes, speeds up security patches through automerging minor updates, and organizes updates for easier review and management.
+
+The `renovate-updates` branch is a dedicated branch where Renovate creates all its update pull requests instead of directly targeting the main development branch (like `main` or `develop`). This setup helps keep automated dependency updates isolated from the main codebase, making it safer and easier for teams to review, test, and merge updates without risking unintended changes to production or active development. It also provides a controlled environment where update PRs can be grouped, rebased, and tested before being manually promoted to the primary branches.
```

Comment:
🤔 this feels like forcing a git-flow like workflow on teams. If you have automated tests, why wouldn't you want to merge updates directly into main? It also makes it more likely that the `renovate-updates` branch _doesn't_ get merged into main, and teams fall behind on updates.